### PR TITLE
T41076 Added Grafana Operator sections

### PR DESCRIPTION
**Pull request by vikram-bitnami**

### CODE REVIEW

**Comment by randradas** [comment]
Hi @vikram-bitnami thank you for your PR, it is expected to have the original template filled with "Description of the change", "Benefits", "Possible drawbacks", "Applicable issues" and "Additional information". Since this is only documentation maybe it would be enough with "Description of the change". Please let me know whether you agree since I know that you are the one in charge of documentation and I'm not pretty sure I'm requesting something uncommon.

**Comment by vikram-bitnami** [comment]
> Hi @vikram-bitnami thank you for your PR, it is expected to have the original template filled with "Description of the change", "Benefits", "Possible drawbacks", "Applicable issues" and "Additional information". Since this is only documentation maybe it would be enough with "Description of the change". Please let me know whether you agree since I know that you are the one in charge of documentation and I'm not pretty sure I'm requesting something uncommon.

@randradas Thanks for the comment. 
- I have already added Description and Additional info in the original PR https://github.com/bitnami/charts/pull/7389 and now further updated it with Benefits. Hope that is fine, let me know in case you need further information.
- For this repo, we don't have any specific format or template for PRs

**Comment by vikram-bitnami** [comment]
> > Hi @vikram-bitnami thank you for your PR, it is expected to have the original template filled with "Description of the change", "Benefits", "Possible drawbacks", "Applicable issues" and "Additional information". Since this is only documentation maybe it would be enough with "Description of the change". Please let me know whether you agree since I know that you are the one in charge of documentation and I'm not pretty sure I'm requesting something uncommon.

As additional context, these PRs are part of a larger task to reduce the overall README size and move examples and large sections to the docs.bitnami.com site. That's why there is no specific description but rather, each README/chart is being processed in sequence and a PR created for each change in both charts and charts-docs repo. Please see T41077 for more info.

File: common/_deploy_extra_resources.md.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+apps: airflow, apache, cassandra, consul, dokuwiki, drupal, etcd, fluentd, ghost, grafana,  harbor, influxdb, jasperreports, jenkins, joomla, jupyterhub, kafka, keycloak, kiam, kong, kubernetes-event-exporter, kubewatch, logstash, magento, mariadb, mediawiki, memcached, minio, moodle, mysql, nats, nginx, nginx-ingress-controller, node, opencart, orangehrm, osclass, owncloud, parse, phabricator, phpbb, phpmyadmin, postgresql, postgresql-ha, prestashop, redis, redis-cluster, redmine, solr, spark, spring-cloud-dataflow, suitecrm, testlink, tomcat, wavefront, wavefront-adapter-for-istio, wavefront-prometheus-storage-adapter, wildfly, wordpress, zookeeper
```
Comment (juan131) [inline common/_deploy_extra_resources.md.erb:3]: Why did you remove it from the list?

Comment (vikram-bitnami) [reply common/_deploy_extra_resources.md.erb:3]: I removed it from the list to avoid a conflict with the custom section with the same id created specifically for the operator chart. It is included in this diff. I used a custom section since there is a specific example for this chart, so the default section is not sufficient.

Comment (juan131) [reply common/_deploy_extra_resources.md.erb:3]: Fair enough, thanks!

**Review by juan131 — APPROVED** [review]
LGTM
